packages/components/src/FormatDate/FormatDate.test.tsx

```diff
@@ -2,17 +2,51 @@ import React from "react";
 import renderer from "react-test-renderer";
 import { cleanup } from "@testing-library/react";
 import { CivilDate } from "@std-proposal/temporal";
-import { FormatDate } from "./FormatDate";
+import { FormatDate, strFormatDate } from "./FormatDate";
 
 afterEach(cleanup);
 
-Object.entries({
-  CivilDate: new CivilDate(2020, 2, 26),
-  ISO8601DateString: "2019-03-30T00:45",
-  Date: new Date("2019-03-30T00:45"),
-}).forEach(([inputType, value]) => {
-  it(`renders a FormatDate from ${inputType}`, () => {
-    const tree = renderer.create(<FormatDate date={value} />).toJSON();
-    expect(tree).toMatchSnapshot();
+describe("Different date values", () => {
+  const dates = {
+    CivilDate: new CivilDate(2020, 2, 26),
+    ISO8601DateString: "2019-03-30T00:45",
+    Date: new Date("2019-03-30T00:45"),
+  };
+
+  const mockDateResult = [`Feb 26, 2020`, `Mar 30, 2019`, `Mar 30, 2019`];
+
+  Object.entries(dates).forEach(([inputType, value], index) => {
+    it(`renders a formatted date from ${inputType}`, () => {
+      const tree = renderer.create(<FormatDate date={value} />).toJSON();
+      expect(tree).toMatchInlineSnapshot(
```

Comment:
Honest question: Can we use `toMatch` instead?  Don't we normally try to avoid snapshots?  What happens if they're inline?

Reply:
ah yup. totally! It basically just returns a string inline.

```jsx
expect(foo).toMatchInlineSnapshot(`"Foo"`)
```